### DISCUSSION TITLE
Release convention

### DISCUSSION THREAD

**Discussion by yohanchatelain** [verificarlo/verificarlo#268] [Ideas]
Hi @verificarlo/devs,
I open a new discussion to define a release convention to avoid confusions like we had for the last release.
This discussion can also be an opportunity to draft a contributing guidelines.
For me, it would be nice if:
- As mentionned by @pablooliveira, we should try to maintain the CHANGELOG when pushing new PR. It would help to not forgot new key features and bugfixes.
- Open a new discussion when we plan a new release where we will expose the CHANGELOG and ask for approval of developers. We could define a short delay after which we would release.

What do you think?

**Comment by pablooliveira** [comment]
Thanks for this proposal which I very much agree with.
Regarding the versioning scheme, we have been loosely following [Semantic Versionning](https://semver.org/) which I think is nice to indicate how much has changed between releases.
